src/dotnet/Chat.UI.Blazor/Components/RightClickChatMenu/RightClickMenuDelete.razor

```diff
@@ -0,0 +1,24 @@
+@namespace ActualChat.Chat.UI.Blazor.Components
+@using BlazorContextMenu
+
+<Item OnClick="@(_ => OnDelete(Chat.Id))" CssClass="context-menu-item warning-item">
```

Comment:
Rename method to `Delete` or replace ad-hoc method with `OnDeleteClick` handler method